File: live/src/main/java/com/hutcwp/live/livebiz/ui/component/publicmessage/psg/holder/HeaderChatHolder.java
package com.hutcwp.live.livebiz.ui.component.publicmessage.psg.holder;

import android.view.View;
import android.widget.TextView;

import com.hutcwp.live.livebiz.ui.component.publicmessage.lib.BaseChatViewHolder;
import com.hutcwp.livebiz.R;

/**
 * 头部信息
 * @author RyanLee
 */
public class HeaderChatHolder extends BaseChatViewHolder {

    public HeaderChatHolder(View itemView) {
        super(itemView);
    }

    @Override
    public void bindData(Object obj, int position) {
        TextView tips = (TextView) getView(R.id.tv_header_text_msg);
        tips.setText(tips.getContext().getResources().getString(R.string.test_healthy_live));
    }
}

File: live/src/main/java/com/hutcwp/live/chat/binder/chat/HeaderChatHolder.kt
package com.hutcwp.live.chat.binder.chat

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.drakeet.multitype.ItemViewBinder
import com.hutcwp.live.R
import com.hutcwp.live.chat.bean.MyChatMsg

/**
 * 头部信息
 * @author RyanLee
 */
class HeaderChatHolder: ItemViewBinder<MyChatMsg, HeaderChatHolder.ViewHolder>() {

    override fun onCreateViewHolder(inflater: LayoutInflater, parent: ViewGroup): ViewHolder {
        return ViewHolder(inflater.inflate(R.layout.live_layout_header_text, parent, false))
    }

    override fun onBindViewHolder(holder: ViewHolder, item: MyChatMsg) {
        holder.tips.setText("健康生活")

    }

    class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
        val tips: TextView = itemView.findViewById(R.id.tv_header_text_msg)
    }

}